packages/api-reference/src/features/ExampleRequest/ExampleRequest.vue

```diff
@@ -113,7 +115,7 @@ async function generateSnippet() {
   }
 
   // Generate a request object
-  const request = getHarRequest(
+  const harRequest = getHarRequest(
```

Comment:
I don't have a good understanding of the differences between `HarRequest` and `Request` types. This will be key for streamlining this api.

Reply:
Not sure if it's possible yet but this `getHarRequest` may be helpful or even replace the `createRequest` helper util

Reply:
I renamed `getHarRequest` to `mergeRequests` to make its intention more clear. :v:

I tried getting rid of HarRequest and using Request only, but I run into some issues. Need to try again tomorrow.